src/components/SecondaryPanes/Breakpoints.css

```diff
@@ -53,18 +53,26 @@ html .breakpoints-list .breakpoint.paused {
   border-color: var(--breakpoint-active-color-hover);
 }
 
-.breakpoints-list .breakpoint-checkbox {
-  margin-inline-start: 0;
-  vertical-align: -2px;
-}
-
 .breakpoints-list .breakpoint-label {
+  max-width: calc(100% - 30px);
   display: inline-block;
   padding-inline-start: 2px;
-  padding-bottom: 4px;
   cursor: default;
 }
 
+.breakpoint-label .breakpoint-checkbox {
+  margin-inline-start: 0;
+  vertical-align: text-bottom;
```

Comment:
Move .breakpoint-checkbox under .breakpoint-label to match the HTML structure.
TBH, using `vertical-align: text-bottom` to control the center vertical alignment look is not general enough when considering if the font size changed in the future. However, it is simple, following the previous way, and quite explicit about we are addressing vertical alignment.